src/constants/ballotStrategies/ballot-strategies.ts

```diff
@@ -0,0 +1,36 @@
+import { constants } from 'ethers'
+import { t } from '@lingui/macro'
+
+export type Strategy = {
+  address: string
+  name: string
+  description?: string
+  unknown?: boolean
+}
+
+export function ballotStrategies() {
```

Comment:
Point 2: Made this a function, and separate where it was being called (getBallotStrategiesByAddress) into a separate file

Reply:
cool, i think it makes more sense to have these in their own file anyway.

for consistency, could we make this filename ballotStrategies.ts?